@xen-orchestra/fs/src/abstract.js

```diff
@@ -409,6 +413,10 @@ export default class RemoteHandlerAbstract {
   // called to finalize the remote
   async _forget(): Promise<void> {}
 
+  async _getInfo() {
+    return Promise.resolve({})
```

Comment:
Why `Promise.resolve()`?

Reply:
Because I had an expected promise mistake. But with your other feedback, it fixed the problem.